rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -207,15 +207,15 @@ namespace rsx
 		{
 			if (rsx->in_begin_end)
 			{
-				rsx->append_array_element(arg & 0xFFFF);
-				rsx->append_array_element(arg >> 16);
+				rsx->append_array_element(se_storage<u16>::swap(arg & 0xFFFF));
+				rsx->append_array_element(se_storage<u16>::swap(arg >> 16));
```

Comment:
swap casts le to be u16 and the function call expands to u32, or do you mean add an explicit static cast?